### PR TITLE
Add service_name, ginkgov2, new deploy process

### PR DESCRIPTION
**Pull request by itdove**
Signed-off-by: Dominique Vernier <dvernier@redhat.com>
https://issues.redhat.com/browse/CMCS-148

add: service_ name
upgrade to ginkgoV2
Fix deployment and functional-test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,16 +22,42 @@ curl http://localhost:8080/metrics
...
 
-It also creates an ingress which allows to retrieve the infomration from outside of the cluster.
+### Prereqs
 
 1. `oc login` your hub RHACM cluster.
-2. Set the image you want to deploy in [deployment.yaml](overlays/deploy/deployment.yaml)
-3. run `make deploy`
-4. Open Prometheus console and check for metrics "acm_managed_cluster_info"
+2. Set in the [kustomization.yaml](./deploy/kustomization.yaml#L6) the namespace where you want the collector to be deployed.
+3. Same for [servicemonitor.yaml](./overlays/deploy/servicemonitor.yaml#L26) and [clusterrole_binding.yaml](./deploy/clusterrole_binding.yaml#L15)
```
Comment (haoqing0110) [inline README.md:33]: what does same for servicemonitor.yaml and clusterrole_binding.yaml? user needs extra steps to modify the 2 files?

Comment (itdove) [reply README.md:33]: Do the same of step 3, means set the namespace.

**Review by haoqing0110 — APPROVED** [review]
/lgtm
The metrics changes looks good to me.

**Pull request merged**